zero_bin/prover/src/lib.rs

```diff
@@ -57,29 +59,29 @@ impl BlockProverInput {
         let block_generation_inputs = self.block_trace.into_txn_proof_gen_ir(
             &ProcessingMeta::new(resolve_code_hash_fn),
             other_data.clone(),
-            prover_params.batch_size,
+            prover_config.batch_size,
         )?;
         info!(
             "Proving block {}, generation inputs count: {}, batch size: {}, segment chunk size: {}",
             block_number,
             block_generation_inputs.len(),
-            prover_params.batch_size,
-            prover_params.segment_chunk_size
+            prover_config.batch_size,
+            prover_config.segment_chunk_size
```

Comment:
nit: maybe lighter to expose the fields above as 

```rust
let ProverConfig {
    max_cpu_len_log,
    batch_size,
    segment_chunk_size,
    save_inputs_on_error
} = prover_config;
```

and then reuse across this function?